.github/workflows/scan_images.yaml

```diff
@@ -0,0 +1,64 @@
+name: Scan Images In Bahmni Namespace
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  scan-images:
+    name: Scan Images In Bahmni Namespace
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
```

Comment:
Checkout is not needed here as this job is not dependant on any files from the current repository